### ISSUE TITLE
Logos for module

### ISSUE THREAD

**Issue by sjib** [teksi/district_heating#65]
![grafik](https://github.com/user-attachments/assets/59b76bad-5567-4db3-b90f-4b849c3df723)
![grafik](https://github.com/user-attachments/assets/f0070582-897b-4800-a71c-57c04edb2f5e)

@jpdupuy These are the logos developed when we started with TEKSI. Are you ok to go along with these or should we adapt something before?

**Comment by jpdupuy** [comment]
@sjib Fine with me, Go for it

**Comment by sjib** [comment]
Yes, I just first wanted to confirm that everyone is ok with that design before.

**Comment by sjib** [comment]
We have all formats on here: https://www.teksi.ch/les-logos-teksi/
I will ask P. to add them for tdh

**Comment by sjib** [comment]
- [ ] ajouter les logos sur github @sjib 
- [ ] ajouter dans la documentation @sjib 
- [ ] adaption nouvelle module sur site TEKSI - https://www.teksi.ch/modules/ @sjib 
- [ ] @jpdupuy Adaption du text descriptif et quelques photos:

```
Le module TEKSI Chauffage à distance est un outil d’aide à la décision qui vous permet de :

    gérer et cartographier les données de votre réseau d’eau potable avec tous ses composants comme par exemple les conduites, les vannes, les pompes, les réservoirs, etc.
    indiquer les caractéristiques des objets du réseau sous forme d’attributs comme le diamètre, le matériau, la profondeur de pose, les fuites, la date d’installation, etc.
    produire des plans et d’extraire des statistiques de la base de données comme la valeur du réseau, la longueur totale des conduites, l’identification des futures interventions, etc.
```

**Comment by jpdupuy** [comment]
Le module TEKSI Chauffage à distance est un outil d’aide à la décision qui vous permet de :

    gérer et cartographier les données de votre réseau de Chauffage à distance avec tous ses composants comme par exemple les conduites, les vannes, les stations de pompages, les chambres, câbles, etc.
    indiquer les caractéristiques des objets du réseau sous forme d’attributs comme le diamètre, le matériau, la profondeur de pose, la date d’installation, etc.

![TDH_form_conduite](https://github.com/user-attachments/assets/2c70ec02-bcc6-4acf-9902-90a18d037728)
![TDH_form_sous_station](https://github.com/user-attachments/assets/9ddd431a-4caa-4e2c-aa57-1f6343e0f5d5)
![TDH_form_structure](https://github.com/user-attachments/assets/72ccc2f1-aa13-46bf-84aa-c232f417497b)
![TDH_form_vanne](https://github.com/user-attachments/assets/504497e0-0694-4373-b18d-21c10e7b5d96)
![TDH_form_alimentation](https://github.com/user-attachments/assets/a42e14f7-dcf5-4317-951e-5458c2e4842a)

![TDH_network_aerial_614x449](https://github.com/user-attachments/assets/de0d2b73-8f2c-43ef-bdfb-b1f09b67f3a5)
